concent_api/core/tests/test_payments_database_transaction_storage.py

```diff
@@ -77,11 +84,12 @@ def test_that_get_all_tx_should_return_list_of_all_transactions(self):
         for transaction in all_transactions:
             self.assertIsInstance(transaction, Transaction)
 
-    def test_that_put_tx_and_inc_nonce_should_create_transaction_and_increase_nonce(self):
+    def test_that_set_nonce_sign_and_save_tx_should_create_transaction_and_increase_nonce(self):
```

Comment:
What does it mean "should create transaction"? I see that transaction is being made "manually" in line "89"

Reply:
Yea, I see it now :) Done